code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -410,18 +410,18 @@
 /obj/item/melee/blood_magic/customised_abstract_text(mob/living/carbon/owner)
 	return "<span class='warning'>[owner.p_their(TRUE)] [owner.l_hand == src ? "left hand" : "right hand"] is burning in blood-red fire.</span>"
 
-/obj/item/melee/blood_magic/attack_self(mob/living/user)
-	afterattack(user, user, TRUE)
+/obj/item/melee/blood_magic/attack_self__legacy__attackchain(mob/living/user)
+	attackby__legacy__attackchain(user, user, TRUE)
```

Comment:
Should this be attackself__legacy_attackchain?

Reply:
I tested it, and not only does it work fine but in hindsight I don't know what I was thinking when I said a proc should call itself.

Reply:
Thanks for taking the time to test it. I had to drop this and circle back to other things but I was absolutely going to.